cmd/funcbench/main.go

```diff
@@ -0,0 +1,261 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func main() {
+	app := kingpin.New(filepath.Base(os.Args[0]), "benchmark result posting and formating tool.\n-i location of github hook file (even.json)")
+	app.HelpFlag.Short('h')
+	input := app.Flag("input", "path to event.json").Short('i').Default("/github/workflow/event.json").String()
+	kingpin.MustParse(app.Parse(os.Args[1:]))
+
+	data, err := ioutil.ReadFile(*input)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	// Temporary fix for the new Github actions time format. This makes the time stamps unusable.
```

Comment:
```suggestion
	// Temporary fix for the new Github actions time format. This makes the time stamps json parsable.
```

Reply:
yes, it makes them parsable, but it also replaces them with a constant, which means that using them is useless. So I am not sure how to deal with this situation.

Reply:
If you are not using it than can merge as is and refactor once they answer your support ticket.